projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```diff
@@ -1,3 +1,17 @@
+/* 
```

Comment:
There is an issue in that Newspack Post carousel block doesn't properly override the next and previous buttons, so the default swiper css gets reapplied.

I believe it is a slower process to get Newspack changes through the ETK release process so have added a temporary fix for this in the carousel css - https://github.com/Automattic/jetpack/pull/20290/files#diff-72023849d8cb93014b6aafc178f8b48eadb46607ad5c9eb6b2a0c419deebf243R1

Our problem is if we do just reset this without the .wp-block-newspack-blocks-carousel selector we may break things for other plugins that use the slider defaults for this. 

The the other alternative is to customise projects/plugins/jetpack/modules/carousel/swiper-bundle.css  to not include it, but that means we need to remember to customise this file with every swiper library update.

Reply:
Can you drop a link to the issue with the Newspack carousel so we can track when it is safe to remove the temporary fix?

Reply:
https://github.com/Automattic/jetpack/issues/20305